src/pipe_mgr/core/dal/dpdk/pipe_mgr_ctx_dpdk_json.c

```diff
@@ -615,6 +630,14 @@ int dal_ctx_json_parse_extern(int dev_id,
 				goto externs_cleanup;
 			}
 
+			ctx->externs_tables_name[ctx->num_externs_tables] =
+				P4_SDE_CALLOC(1, P4_SDE_TABLE_NAME_LEN);
+			if (!ctx->externs_tables_name) {
```

Comment:
we may need to do NULL check as  ctx->externs_tables_name[] ? and also need to free "ctx->externs_tables_name" in case of failure

Reply:
NULL check is taken care at line 635, will take care of free for failure cases

Reply:
taken care NULL check and memory free